.github/workflows/circle_artifacts.yml

```diff
@@ -1,4 +1,4 @@
-on: [status]
+on: [status]  # yamllint disable-line rule:truthy
```

Comment:
Fun fact -- "on" is truthy and actually you can have truthy keys in YAML. So a slightly more correct alternative here would be to change this to `"on": ...`. But this isn't how I've ever seen a `.github/workflows` example and it's not how they do it in their docs. So it seemed like the safest option was an inline ignore. See also https://github.com/adrienverge/yamllint/issues/430